go.mod

```diff
@@ -23,8 +23,8 @@ require (
 	github.com/lightninglabs/lndclient v0.20.0-1
 	github.com/lightninglabs/loop/looprpc v1.0.7
 	github.com/lightninglabs/loop/swapserverrpc v1.0.14
-	github.com/lightninglabs/taproot-assets v0.7.0-rc1.0.20250925100956-c44e078c33e0
-	github.com/lightninglabs/taproot-assets/taprpc v1.0.10-0.20250925100956-c44e078c33e0
+	github.com/lightninglabs/taproot-assets v0.7.0-rc1.0.20251014172227-e6ae082c0b4b
```

Comment:
:+1: 
We'll soon tag our rc2 so eventually you might want to bump again to a tagged version, if preferred.